### DISCUSSION TITLE
Quill-Mention compatibility

### DISCUSSION THREAD

**Discussion by pr4ctic** [slab/quill#4062] [Q&A]
Hey!
Back when there was only v1.3.7 I was able to use quill-mention with no issue. Now it seems like on v2.0 every time I paste text it gets automatically "cleaned". 
So, on v1.3.7, when I created a mention, it looked like this:
`<p>My paragraph can be found on <span class="mention" data-denotation-char="" data-id="1" data-value="www.google.cl">www.google.cl</span></p>`

Now, on v2, it gets transformed into 
`<p>My paragraph can be found on www.google.cl</p>`

So, basically I lose my mention. ¿Is there any way to prevent this behavior?
